src/replication.c

```diff
@@ -1363,12 +1375,21 @@ int istgt_lu_create_snapshot(spec_t *spec, char *snapname, int io_wait_time, int
 	rcomm_mgmt->caller_gone = 1;
 	if (rcomm_mgmt->cmds_sent == (rcomm_mgmt->cmds_succeeded + rcomm_mgmt->cmds_failed)) {
 		free_rcomm_mgmt = 1;
-		if ((rcomm_mgmt->cmds_succeeded == spec->replication_factor) && (rcomm_mgmt->cmds_failed == 0)) {
+		if (rcomm_mgmt->cmds_succeeded >= cf) {
```

Comment:
this can be moved out of the 'if' condition (cmds_sent == cmds_succeeded + cmds_failed)

Reply:
done.